### PR TITLE
Remove error when trying to rm db

### PR DESCRIPTION
**Pull request by juliaogris**
Resolves part of #105.

Changes proposed in this pull request:
- Remove TodoNagger experiment (@anzdaddy 's suggestion)
- Close database connection before trying to remove file

@anz-bank/sysl-developers

### CODE REVIEW

File: src/sysl/reljam/__main__.py [hunk 1]
```diff
@@ -5,5 +5,6 @@
 from sysl.reljam import reljam
 
 if __name__ == '__main__':
+    "asdlkfjalsdkfj"
```
Comment (anzdaddy) [inline src/sysl/reljam/__main__.py]: ?

File: src/sysl/util/cache.py [hunk 2]
```diff
@@ -55,11 +55,11 @@ def connection(self):
             if db_ver == schema_ver:
                 break
 
-            self.conn = None
+            self.conn.close()
             try:
                 os.remove(DBPATH)
             except BaseException:
-                print '=== Cannot remove db:', DBPATH
+                print 'Cannot remove db:', DBPATH
```
Comment (anzdaddy) [inline src/sysl/util/cache.py]: `'Warning: Cannot remove …'`

Comment (juliaogris) [reply src/sysl/util/cache.py]: Done. Please re-approve @anzdaddy

File: src/sysl/reljam/__main__.py (hunk 1 above)
Comment (juliaogris) [reply src/sysl/reljam/__main__.py]: oups. Removed. Now ready for review.

**Pull request merged**